.github/workflows/vizro-qa-notebooks-tests-trigger.yml

```diff
@@ -0,0 +1,36 @@
+name: Vizro qa notebooks tests trigger
+
```

Comment:
Given they're almost identical, is there any way to combine these two workflows into one? e.g. with a `matrix` that goes over the two different values.

If this is just a bad idea for performance reasons then please can you add a comment to both files explaining that they're the same apart from XXX, and the reason they are done as separate workflows is YYY.

Reply:
Found a way. Done